addons/mail/static/src/messaging/entity/activity/activity.js

```diff
@@ -67,16 +70,24 @@ function ActivityFactory({ Entity }) {
          * @param {string} param0.feedback
          * @returns {Object}
          */
-        async markAsDoneAndSchedule({ feedback }) {
+        async markAsDoneAndScheduleNext({ feedback }) {
             const action = await this.env.rpc({
                 model: 'mail.activity',
                 method: 'action_feedback_schedule_next',
                 args: [[this.id]],
                 kwargs: { feedback },
             });
-            this.chatter.refresh();
+            const chatter = this.chatter;
+            if (chatter) {
+                this.chatter.refresh();
+            }
             this.delete();
-            return action;
+            this.env.do_action(action, {
+                on_close: async() => {
+                    // If we don't wait for it, errors are not propagated.
```

Comment:
I don't really get this. Wouldn't this generate a unhandled promise rejection error at least??

Reply:
Seems that finally the error is propagated, don't know why it wasn't the case when I had to debug this ..